### PR TITLE
Unit tests for UserServiceImpl

### PR DESCRIPTION
**Pull request by eyouu**

### CODE REVIEW

File: core/src/test/java/greencity/service/impl/UserServiceImplTest.java [hunk 1]
```diff
@@ -737,4 +706,165 @@ public void getActivatedUsersAmountTest() {
...
+    }
+
+    @Test
+    void updateUserProfilePictureTest() {
+        MultipartFile image = new MockMultipartFile("data", "filename.txt", "text/plain", "some xml".getBytes());
+        when(userRepo.findByEmail(anyString())).thenReturn(Optional.of(user));
+
+        try {
+            when(fileService.upload(image)).thenReturn(new URL("http://test.com"));
+        } catch (MalformedURLException e) {
```
Comment (antonbakalets) [inline core/src/test/java/greencity/service/impl/UserServiceImplTest.java:731]: Do not catch exceptions in unit test, just add it to method signature.

File: core/src/test/java/greencity/service/impl/UserServiceImplTest.java [hunk 2]
```diff
@@ -737,4 +706,165 @@ public void getActivatedUsersAmountTest() {
...
+        }
+
+        userService.updateUserProfilePicture(image, anyString());
+        verify(userRepo).save(user);
+    }
+
+    @Test
+    void updateUserProfilePictureNotUpdatedExceptionTest() {
+        when(userRepo.findByEmail(anyString())).thenReturn(Optional.of(user));
+        String email = anyString();
```
Comment (antonbakalets) [inline core/src/test/java/greencity/service/impl/UserServiceImplTest.java:742]: Inialize with test data, I mean some real String, and use it also in line 741

File: core/src/test/java/greencity/service/impl/UserServiceImplTest.java [hunk 3]
```diff
@@ -737,4 +706,165 @@ public void getActivatedUsersAmountTest() {
...
+            userService.getSixFriendsWithTheHighestRating(1L)
+        );
+    }
+
+    @Test
+    void getSixFriendsWithTheHighestRatingTest() {
+        List<UserProfilePictureDto> list = new ArrayList<>();
+        UserProfilePictureDto e = new UserProfilePictureDto();
+        list.add(e);
+        when(userRepo.getSixFriendsWithTheHighestRating(anyLong())).thenReturn(Collections.singletonList(user));
```
Comment (antonbakalets) [inline core/src/test/java/greencity/service/impl/UserServiceImplTest.java:795]: Collections.singletonList(e)

File: core/src/test/java/greencity/service/impl/UserServiceImplTest.java (hunk 1 above)
Comment (eyouu) [reply core/src/test/java/greencity/service/impl/UserServiceImplTest.java]: Fixed

**Pull request merged**